torchao/quantization/quantize_/workflows/float8/float8_semi_sparse_tensor.py

```diff
@@ -0,0 +1,293 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from typing import List, Optional
+
+import torch
+from torch.utils._python_dispatch import return_and_correct_aliasing
+
+from torchao.float8.inference import (
+    FP8Granularity,
+)
+from torchao.ops import (
+    rowwise_scaled_linear_sparse_cutlass_f8f8,
+    to_sparse_semi_structured_cutlass_sm9x_f8,
+)
+from torchao.quantization.granularity import PerRow
+from torchao.quantization.quant_primitives import (
+    _choose_scale_float8,
+    _quantize_affine_float8,
+)
+from torchao.quantization.quantize_.common import (
+    _choose_quant_func_and_quantize_tensor,
+)
+from torchao.quantization.utils import get_block_size
+from torchao.utils import (
+    TorchAOBaseTensor,
+    is_sm_at_least_90,
+)
+
+__all__ = [
+    "Sparse2x4Float8Tensor",
+]
+
+aten = torch.ops.aten
+
+
+from .float8_packing_format import Float8TensorPackingFormat
+from .float8_tensor import QuantizeTensorToFloat8Kwargs
+
+
+class Sparse2x4Float8Tensor(TorchAOBaseTensor):
```

Comment:
the way we structure these is one to one correspondance between packing format and tensors actually, for example: https://github.com/pytorch/ao/blob/8806b02e4e6b89564b6036c8cefc5f16445f3be0/torchao/quantization/quant_api.py#L895-L934

Reply:
ok ill just make this the CUTLASS format and open a new PR for cuSPARSELt .